DMCompiler/Compiler/DM/DMParser.cs

```diff
@@ -366,7 +366,19 @@ public partial class DMParser : Parser<Token> {
                     if (pathElement != null) {
                         if(pathElement == "operator") {
                             Token operatorToken = Current();
-                            if(Check(OperatorOverloadTypes)) {
+                            if(Current().Type == TokenType.DM_Slash) {
+                                //Up to this point, it's ambiguous whether it's a slash to mean operator/(), like the division operator overload
+                                //or "operator" just being used as a normal type name, as in a/operator/b/c/d
+                                Token peekToken = Advance();
+                                if (peekToken.Type == TokenType.DM_LeftParenthesis) { // Disambiguated as an overload
+                                    operatorFlag = true;
+                                    pathElement += operatorToken.PrintableText;
+                                } else { //Otherwise it's just a normal path, resume
+                                    ReuseToken(operatorToken);
+                                    Warning("Using \"operator\" as a path element is ambiguous", Current()); // NOTE: Maybe should be SoftReservedKeyword? Unsure!
```

Comment:
Any opinions on marking this as SoftReservedKeyword? That'd be an error in the default config.

Reply:
SoftReservedKeyword is better than a warning, but I don't know why we'd emit anything unless it impacts behavior

Reply:
This can be spooky for behaviour, since `operator/()` has a few places of grammatical ambiguity in the language. I won't be completely surprised if something somewhere accidentally relies BYOND resolving this vexing parse one way instead of the other. A warning is deserved.